tests/App/Entity/Gallery.php

```diff
@@ -23,16 +23,15 @@
 class Gallery extends BaseGallery
 {
     /**
+     * @var int|null
+     *
      * @ORM\Id
      * @ORM\GeneratedValue
      * @ORM\Column(type="integer")
      */
     protected $id;
 
-    /**
-     * Get id.
-     */
-    public function getId(): int
+    public function getId()
```

Comment:
Can we keep return type hints here? And in other files too?
```suggestion
    public function getId(): ?int
```

Reply:
We can't, on master that typehint is removed because the getId() is a method that comes from the interface and it can accept both int/string.